roles/ceph-mon/tasks/crush_rules.yml

```diff
@@ -11,7 +11,28 @@
     - create_crush_tree
     - hostvars[item]['osd_crush_location'] is defined
 
-- name: create configured crush rules
+- name: create crush ec profiles
+  command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} osd erasure-code-profile set {{ item.name }} plugin={{ item.plugin }} {{ item.ec_config | to_json | regex_replace('\\:\\ ','=') | regex_replace('[\\[\\]{}\\\"]') | regex_replace('\\,\\ ',' ') }}"
```

Comment:
I usually try to avoid using such regex imbrication.
I think that when we need to manage with "complex" yaml data structure in ansible, it's a signal we should maybe create a specific ansible module.